x-pack/plugins/ml/server/models/data_recognizer/modules/network/ml/rare_destination_country.json

```diff
@@ -0,0 +1,35 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "A machine learning job detected an unusual destination country name in the network logs. This can be due to initial access, persistence, command-and-control, or exfiltration activity. For example, when a user clicks on a link in a phishing email or opens a malicious document, a request may be sent to download and run a payload from a server in a country which does not normally appear in network traffic or business work-flows. Malware instances and persistence mechanisms may communicate with command-and-control (C2) infrastructure in their country of origin, which may be an unusual destination country for the source network.",
+  "groups": [
+    "security",
+    "network"
+  ],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "rare by \"destination.geo.country_name\"",
+        "function": "rare",
+        "by_field_name": "destination.geo.country_name",
+        "detector_index": 0
+      }
+    ],
+    "influencers": [
+      "destination.geo.country_name",
+      "destination.as.organization.name",
+      "source.ip",
+      "destination.ip"
+    ]
+  },
+  "allow_lazy_open": true,
+  "analysis_limits": {
+    "model_memory_limit": "32mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp"
+  },
+  "custom_settings": {
+    "created_by": "ml-module-security-windows"
```

Comment:
Looks like this has been copied from the Windows module. It should correspond to the ID of this module, so probably `ml-module-security-network`